src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -71,6 +71,20 @@ namespace UiaOperationAbstraction
         const char* what() const override { return "RemoteOperationsWrapper local loop break"; }
     };
 
+    class UiaFailure

```

Comment:
Can we add a comment to this saying it must only be used inside a catch handler, since it could cause a crash outside of one? If you want we can also make the constructor private, delete copy/move constructors, and make `UiaOperationDelegator` a friend class so only we're able to construct one of these, but feel free to not do that if you think it's unnecessary.